ci-operator/config/redhat-openshift-ecosystem/openshift-preflight/redhat-openshift-ecosystem-openshift-preflight-main__4.8.yaml

```diff
@@ -30,7 +29,7 @@ images:
     RUN curl -Lo /usr/local/bin/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk_linux_${ARCH} \
       && chmod 755 /usr/local/bin/operator-sdk
 
-    RUN curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OPENSHIFT_CLIENT_VERSION}/openshift-client-linux-${OPENSHIFT_CLIENT_VERSION}.tar.gz | tar -xzv -C /usr/local/bin oc
```

Comment:
@acornett21 We should keep these versions pinned, IMO. 

These directly correlate with what Preflight builds into the Container Image it releases. https://github.com/redhat-openshift-ecosystem/openshift-preflight/blob/main/Dockerfile#L45-L46

IIRC, The goal here is to replicate the same environment we use in our release assets.

Reply:
I agree that we should keep dependencies in sync, they weren't even in sync before this PR, but with that said, we are removing the `oc` dependency from `preflight` and can remove it from this automation as well.

Reply:
+1 okay to get rid of the OC client if we don't need it in the rest of the script. Just pin the OpSDK and we should be set to go.